polaris-react/src/components/OptionList/components/Option/Option.tsx

```diff
@@ -99,43 +94,25 @@ export function Option({
     active && styles.active,
     select && styles.select,
     verticalAlign && styles[variationName('verticalAlign', verticalAlign)],
-    polarisSummerEditions2023 && allowMultiple && styles.CheckboxLabel,
-    polarisSummerEditions2023 && allowMultiple && styles.MultiSelectOption,
+    allowMultiple && styles.CheckboxLabel,
+    allowMultiple && styles.MultiSelectOption,
   );
 
-  const checkBoxRole = role === 'option' ? 'presentation' : undefined;
-
   const optionMarkup = allowMultiple ? (
     <label htmlFor={id} className={multiSelectClassName}>
       <div className={styles.Checkbox}>
-        {polarisSummerEditions2023 ? (
-          <PolarisCheckbox
-            id={id}
-            label=""
-            ariaDescribedBy={`${id}-label`}
-            value={value}
-            checked={select}
-            disabled={disabled}
-            onChange={handleClick}
-          />
-        ) : (
-          <Checkbox
```

Comment:
Since we're no longer using the custom checkbox (or exporting it from root), its also deleted in this PR.